### PR TITLE
font: complete migration of font subpackage to separate repo

### PR DESCRIPTION
**Pull request by deadprogram**
This PR completes the migration of the `font` subpackage to separate `tinygl-font` repo.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,10 +1,10 @@
 module github.com/aykevl/tinygl
 
-go 1.20
+go 1.22.0
```
Comment (aykevl) [inline go.mod]: Is this intentional?

File: listbox.go [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package tinygl
 
 import (
-	"github.com/aykevl/tinygl/font"
 	"tinygo.org/x/drivers/pixel"
+	font "tinygo.org/x/tinygl-font"
```
Comment (aykevl) [inline listbox.go:5]: This works just as well (the package name is `font`):

```suggestion
	"tinygo.org/x/tinygl-font"
```

But the existing code is fine too if you prefer it for clarity.

Comment (deadprogram) [reply listbox.go:5]: Seems like `go fmt` wants to change it to be that way? My editor keeps changing it to be how it is in the PR.

File: go.mod (hunk 1 above)
Comment (deadprogram) [reply go.mod]: Running `go mod tidy` keeps changing to that, even if I delete the `go.sum`.

Comment (deadprogram) [reply go.mod]: I see why. Created https://github.com/tinygo-org/tinygl-font/pull/4 to address this.

Comment (deadprogram) [reply go.mod]: Fixed.

**Comment by deadprogram** [comment]
This PR is ready @aykevl

**Comment by deadprogram** [comment]
Reminder to @aykevl please :smile:

File: listbox.go (hunk 2 above)
Comment (aykevl) [reply listbox.go:5]: I suppose it does that for clarity/readability.

**Review by aykevl — APPROVED** [review]

**Pull request merged**